### PR TITLE
Mb/fb tags

### PR DESCRIPTION
**Pull request by MostafaBalata**

### CODE REVIEW

File: page/partials/head.ejs [hunk 1]
```diff
@@ -7,15 +7,17 @@
     <meta name="keywords" content="neufund, ico, neumark, venture capital, ethereum, startup, blockchain" />
     <meta name="robots" content="/robots.txt" />
 
-    <meta property="og:title" content="Neufund |  <%= title %>" />
+    <meta property="og:title" content="Neufund | <%= title %>" />
     <meta property="og:image" content="assets/social_logo.png" />
-    <meta property="og:url" content="http://neufund.org/" />
+    <meta property="og:url" content="https://commit.neufund.org" />
+    <meta property="og:type" content="website" />
+    <meta property="og:description" content="Neufund’s vision is to invite everyone to join the bootstrapping of a new two-sided marketplace and an investment ecosystem, and to once and for all democratise innovation and creation." />
```
Comment (krzkaczor) [inline page/partials/head.ejs]: could you export description that is repeated few time in this file to content.yml?

Comment (MostafaBalata) [reply page/partials/head.ejs]: Good idea

File: page/product.ejs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-    <%- include('partials/head', {title: 'Product' }); %>
+    <%- include('partials/head', {title: 'Product', meta: content.meta_head }); %>
     <body>
```
Comment (krzkaczor) [inline page/product.ejs]: i think you forgot about faq page

Comment (MostafaBalata) [reply page/product.ejs]: No, I didn't 
https://github.com/Neufund/commit.neufund.org/pull/53/files#diff-8d814f366939e25c10181bd637bc829dR3

**Pull request merged**